drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/window/WindowResizer.java

```diff
@@ -70,17 +70,19 @@ private void resizeWindow(DroneEvent event) {
         WebDriverConfiguration configuration = context.get(dronePoint).getConfigurationAs(WebDriverConfiguration.class);
         Validate.stateNotNull(configuration, "WebDriver configuration must not be null");
 
-        if (configuration.getDimensions() != null) {
-            String dimensions = configuration.getDimensions().toLowerCase().trim();
-            Matcher m = DIMENSIONS_PATTERN.matcher(dimensions);
+        String browser = configuration.getBrowser().toLowerCase();
+        if(!browser.equals("chromeheadless")) {
```

Comment:
Webdriver probably doesn't support automatic resizing for chrome but it does for PhantomJs and HtmlUnit. Also, seems like PhantomJs and HtmlUnit have some defaults set for the maximized mode. 
When I try executing our UI tests against these browsers. The values I get for maximized dimension are:
`PhantomJs - (1366, 768)`
`HtmlUnit -(1272, 698)`

Also, I read somewhere - 
` PhantomJS driver, by default, it will run in 400×300. You may want to run it in a desktop resolution. Don’t dispare. It is easy. You can tell Geb driver to maximize its window and it will run in  1366, 768`